api/rest/coordinator.go

```diff
@@ -0,0 +1,256 @@
+package rest
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const numberOfWorkers = 10
+const contextDoneErrMsg = "bundle creation context finished before bundle creation finished"
+
+// BundleStatus tracks the status of local bundle creation requests
+type BundleStatus struct {
```

Comment:
I think this could be package private 
```suggestion
type bundleStatus struct {
```